src/utils/utils.js

```diff
@@ -1,25 +1,37 @@
 export function getDealtCard(G) {
   let dealtCard = "";
   if (G.dealt.length > 0) {
-      dealtCard = G.dealt[G.dealt.length - 1];
+    dealtCard = G.dealt[G.dealt.length - 1];
   }
   return dealtCard;
 }
 
 export function getDealtCardsForPlayers(order, dealt) {
   let cards = Array(order.length).fill("");
-  for (let i=0; i<dealt.length; i++) {
+  for (let i = 0; i < dealt.length; i++) {
     let idx = parseInt(order[i]);
     cards[idx] = dealt[i];
   }
   return cards;
 }
 
+export function isGameModeCornucopia(gameMode) {
+  return (gameMode === "OWASP Cornucopia");
```

Comment:
Maybe it's a good idea to introduce constants like
```js
const GAMEMODE_EOP = "Elevation of Privilege";
const GAMEMODE_CORNUCOPIA = "OWASP Cornucopia";
```
in `constants.js`

Then one could use them consistently throughout the code (i.e. in the dopdowns, in this function etc.), to nt have these magic strings.